docs/AUTH0.md

```diff
@@ -0,0 +1,34 @@
+# Auth0
```

Comment:
The doc file is not linked currently from the main or setup doc right? Would be helpful though I guess